snapcraft/internal/libraries.py

```diff
@@ -73,9 +73,10 @@ def _get_system_libs():
         lib_path = None
 
     if not lib_path or not os.path.exists(lib_path):
-        logger.debug('No libraries to exclude from this release')
-        # Always exclude libc.so.6
-        return frozenset(['libc.so.6'])
+        logger.debug('Only excluding libc libraries from the release')
+        libc6_libs = [os.path.basename(l)
+                      for l in repo.Repo.get_package_libraries('libc6')]
```

Comment:
`get_package_libraries` definitely includes more than just libs. Will that cause issues, here?